### PR TITLE
configuration file

### PR DESCRIPTION
**Pull request by Shilpa-Manjunath**

### CODE REVIEW

File: admin/kernel-masterdata-service/docs/Configuration.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* `mosip.admin.ui.configsversion=version:${application.configuration.level.version};HierarchyLevel:${mosip.recommended.centers.locCode}`
+
+## Value limit on fields
+* `mosip.kernel.registrationcenterid.length:specify length of registration center id`
+* `mosip.kernel.machineid.length:specify length of machine id`
+* `mosip.recommended.centers.locCode:specify hierarchy location code`
+* `master.search.maximum.rows:specify maximum number of rows to be fetched in the search and filter queries`
+* `masterdata.registerdevice.timestamp.validate:specify value with + or - to validate device time stamp`
+* `application.configuration.level.version:specify version of application`
+* `mosip.kernel.applicant.type.age.limit:specify adult age limit this differs based on country`
```
Comment (ase-101) [inline admin/kernel-masterdata-service/docs/Configuration.md:33]: mosip.kernel.applicant.type.age.limit is this property still in use ?

File: admin/kernel-masterdata-service/docs/Configuration.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* `masterdata_database_password`
+* `masterdata_database_url`
+
+Point the above to your DB .  Default is set to point to in-cluster Postgres installed with sandbox.
+
+## Admin UI config
+* `mosip.admin.ui.configs`
+Above property is used for UI configuration of admin. Value of the property is a key value pairs. Example:
+
+* `mosip.admin.ui.configsversion=version:${application.configuration.level.version};HierarchyLevel:${mosip.recommended.centers.locCode}`
```
Comment (ase-101) [inline admin/kernel-masterdata-service/docs/Configuration.md:24]: it is important to mention about language-related properties which is important for admin-UI

File: admin/kernel-masterdata-service/docs/Configuration.md (hunk 1 above)
Comment (Shilpa-Manjunath) [reply admin/kernel-masterdata-service/docs/Configuration.md:33]: Earlier this  was used in mvel file ,cross verified ,it is not used in mvel file now logic is changed will remove this property

File: admin/kernel-masterdata-service/docs/Configuration.md (hunk 2 above)
Comment (Shilpa-Manjunath) [reply admin/kernel-masterdata-service/docs/Configuration.md:24]: This is just an example so which ever properties are required for UI Configuration those needs to be added as a part of configs

File: admin/admin-service/src/main/java/io/mosip/admin/config/Mapper.java [hunk 3]
```diff
@@ -156,7 +156,7 @@ private static void loadEntity() {
 		entityMap.put("doc_category", DocumentCategory.class);
 		entityMap.put("doc_type", DocumentType.class);
 		entityMap.put("dynamic_field", DynamicField.class);
-		entityMap.put("reg_exceptional_holiday", ExceptionalHoliday.class);
+		//entityMap.put("reg_exceptional_holiday", ExceptionalHoliday.class);
```
Comment (ase-101) [inline admin/admin-service/src/main/java/io/mosip/admin/config/Mapper.java:159]: Why is this removed ?

Comment (Shilpa-Manjunath) [reply admin/admin-service/src/main/java/io/mosip/admin/config/Mapper.java:159]: two entries with same key different values, internally first value will be replaced with reassigned value hence commented this.

**Review by ase-101 — APPROVED** [review]
